README.md

```diff
@@ -175,46 +184,50 @@ After downloading/installing the prerequisites, ensure that you can execute the
 3. On Windows: `where sh`
 ```
 
-To run Zowe CLI system tests, you need a configured properties file with proper system information present.
+To run Zowe CLI system tests, you need a configured properties file populated with proper system information.
 
-A dummy properties file is present in the `__tests__/__resources__/properties folder`, `default_properties.yaml`. Using this file as a template, you should create a `custom_properties.yaml` file within the same directory. Git is configured to ignore all properties files in the properties folder, except for the `default_properties.yaml` file. If the `custom_properties.yaml` file cannot be found or loaded, an error with relevant details will be thrown when attempting to run tests.
+A dummy properties file is available in the `default_properties.yaml` file **[is this correct? I couldn't find it in the repo]** in the `__tests__/__resources__/properties` folder. Using this file as a template, you should create a `custom_properties.yaml` file within the same directory. Git is configured to ignore all properties files in the properties folder, except for the `default_properties.yaml` **[is this correct? I couldn't find it in the repo]** file. If the `custom_properties.yaml` file cannot be found or loaded, an error with relevant details displays when attempting to run tests.
 
-You can then run the system tests by issuing the following command:
+Run the system tests:
 
 ```
 npm run test:system
 ```
 
 <br/>
 
-**IMPORTANT!** Do not commit configured properties files because they contain security principles and other critical information.
+**IMPORTANT!** Do not commit configured properties files to your repository **[is "to your repo" correct here?]** because they contain security principles and other critical information.
```

Comment:
@ATorrise @adam-wolfe - please respond

Reply:
I would say "Do not commit configured properties files to _this repository_ because..."

The risk here is that somebody contributing to Zowe CLI might inadvertently add a properties file that could contain user/password or other sensitive information that are needed for testing something a mainframe.